client/lib/analytics/ad-tracking.js

```diff
@@ -276,7 +276,11 @@ function loadTrackingScripts( callback ) {
  * @returns {Boolean}
  */
 function isAdTrackingAllowed() {
-	return config.isEnabled( 'ad-tracking' ) && ! doNotTrack() && ! isPiiUrl();
+	// TODO: Remove the following line before release.
+	return ! shouldSkipAdsForThisUrl() && config.isEnabled( 'ad-tracking' ) && ! doNotTrack() && ! isPiiUrl();
```

Comment:
Do we need two different functions to check the URL? We have `shouldSkipAdsForThisUrl` and `isPiiUrl`, could we combine these to make this conditional shorter? We could call `isPiiUrl` from `shouldSkipAdsForThisUrl`...

Reply:
I like this idea. Note that `isPiiUrl` is used in two places: Checking if we should perform analytics and if we should inject ad scripts. 

Let me whip something up so that we would have two functions: `shouldSkipAds()` and `shouldSkipAnalytics()`.